documentation/asciidoc/topics/ref_cacheservice_resources.adoc

```diff
@@ -44,3 +45,4 @@ spec:
 <8> Allocates resources to nodes.
 <9> Configures logging.
 <10> Configures services for external traffic.
+<11> Configures anti-affinity strategies that guarantee {brandname} cluster availability.
```

Comment:
FYI it can also be used to configure affinity strategies, which is useful if you want to ensure that your cluster pods are co-located on the same node as an application  pod for greater performance.